vyper/ovm/transpile_lll.py

```diff
@@ -189,10 +189,9 @@ def _with_subroutine_vars(self, subroutine):
         def build(evm_args, subroutine):
             if len(evm_args) == 0:
                 return subroutine
-            else:
-                arg = evm_args[0]
-                remaining_args = evm_args[1:]
-                return ["with", arg, "pass", build(remaining_args, subroutine)]
+            arg = evm_args[0]
+            remaining_args = evm_args[1:]
+            return ["with", arg, "pass", build(remaining_args, subroutine)]
```

Comment:
Keep this one as is, it's a little easier to see that two branches exist (and only two) when it's indented in an if statement

Reply:
fixed